--- md-validator-client.gemspec.orig	2021-08-31 10:25:33.000000000 +0100
+++ md-validator-client.gemspec	2021-08-31 10:25:56.000000000 +0100
@@ -29,7 +29,7 @@
 
   s.add_runtime_dependency 'typhoeus', '~> 1.0', '>= 1.0.1'
   s.add_runtime_dependency 'json', '~> 2.1', '>= 2.1.0'
-  s.add_runtime_dependency 'addressable', '~> 2.3', '>= 2.3.0'
+  s.add_runtime_dependency 'addressable', '~> 2.3', '>= 2.3.8'
 
   s.add_development_dependency 'rspec', '~> 3.6', '>= 3.6.0'
   s.add_development_dependency 'vcr', '~> 3.0', '>= 3.0.1'
